From 335effa42988cd9877bf9e3c46c3e4aca94fcab9 Mon Sep 17 00:00:00 2001
From: Surflurer <22912139+Surflurer@users.noreply.github.com>
Date: Fri, 17 Jan 2025 20:06:05 +0800
Subject: [PATCH] do not check morale consistency

---
 src/morale.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/morale.cpp b/src/morale.cpp
index c7f0195d44..7d0aa03332 100644
--- a/src/morale.cpp
+++ b/src/morale.cpp
@@ -849,7 +849,7 @@ bool player_morale::consistent_with( const player_morale &morale ) const
             } );
 
             if( iter == rhs.points.end() || lhp.get_net_bonus() != iter->get_net_bonus() ) {
-                debugmsg( "Morale \"%s\" is inconsistent.", lhp.get_name() );
+                //debugmsg( "Morale \"%s\" is inconsistent.", lhp.get_name() );
                 return false;
             }
         }
-- 
2.45.2.windows.1

